tests/test_transforms.py

```diff
@@ -39,13 +40,14 @@ def test_q_to_tth(wavelength, q, expected_tth, wavelength_warning_msg):
 @pytest.mark.parametrize(
     "wavelength, q, expected_error_type",
     [
-        # UC1: user specified invalid q values that result in tth > 180 degrees
+        # Test error messages in q to tth conversion with invalid Two theta values.
+        # C1: Invalid q values that result in tth > 180 degrees
         (
             4 * np.pi,
             np.array([0.2, 0.4, 0.6, 0.8, 1, 1.2]),
             ValueError,
         ),
-        # UC2: user specified a wrong wavelength that result in tth > 180 degrees
```

Comment:
a comment starting with `user specified...` feels a bit repetitive to me and does not add great value..